### PR TITLE
feat: splitting fossa-scan and private semantic action (#131)

### PR DESCRIPTION
**Pull request by uoboda-splunk**

### CODE REVIEW

File: .github/workflows/reusable-escu-manual-workflow.yml [hunk 1]
```diff
@@ -128,7 +130,7 @@ jobs:
         with:
           submodules: recursive
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```
Comment (kkania-splunk) [inline .github/workflows/reusable-escu-manual-workflow.yml:133]: is it correct?

Comment (kkania-splunk) [reply .github/workflows/reusable-escu-manual-workflow.yml:133]: I see such branch exists

Comment (uoboda-splunk) [reply .github/workflows/reusable-escu-manual-workflow.yml:133]: I believe yes, discussed it with @mkolasinski-splunk Can you confirm?

**Review by kkania-splunk — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "devDependencies": {
+    "@semantic-release/changelog": "^6.0.2",
```
[the remark this answers is not in the document]
Comment (uoboda-splunk) [reply package.json:3]: I will trigger another release. We missed one PR in semantic-release action

**Pull request merged**